app/serializers.py

```diff
@@ -0,0 +1,42 @@
+import json
+import xml.etree.ElementTree as ET  # noqa: N817
+from abc import ABC, abstractmethod
+
+
+class Serializer(ABC):
+    @staticmethod
+    @abstractmethod
+    def serialize(title: str, content: str) -> str:
+        pass
+
+
+class JSONSerializer(Serializer):
+    @staticmethod
```

Comment:
A staticmethod has no access to the instance (self) or class (cls). This can be limiting if, in the future, you want to serialize data based on instance-specific attributes or configurations. For instance, a JSONSerializer instance could have its own configuration that alters how it serializes data. By making it a static method, you prevent the class from evolving in this way.